examples/MeteorCLI/all-in-one/imports/polyfills.ts

```diff
@@ -1,5 +1,14 @@
-import 'zone.js';
 import 'core-js/es7/reflect';
+const oldThen = Promise.prototype.then;
+Object.defineProperty(Promise.prototype, "then", {
```

Comment:
@ardatan could you add a comment pointing out the right issues and PRs on zone.js and Meteor?